src/lib/capabilities.tsx

```diff
@@ -134,6 +137,10 @@ export const getLWCapabilities = (
     accessEditor: true,
     /* menu */
     formatArticle: !isViewer(),
+    /* editor */
+    editorWrite: !isWriter(),
```

Comment:
Isn't the point of that story to stray away from using internal manuscripts role for permissions resolving? That's to say I would expect to get specific permissions from WM for that. Is that not right?

Reply:
I don't think so, will move role-checking from editor to capabilities, to be in one place. @mnatsheh please confirm if that is true. 

and the permissions added here will replace roles checking in the editor